Datasnap/Classes/Utilities/Datasnap.m

```diff
@@ -124,9 +130,16 @@ - (void)initializeData
         self.identifier.mobileDeviceGoogleAdvertisingIdOptIn = @"NO";
     }
     self.identifier.globalDistinctId = [[NSUUID UUID] UUIDString];
+    self.identifier.hashedEmail = self.email;
```

Comment:
This is where you would initialize a hashedEmail property with an actual hash once you turn the sha1 method into a category helper. `self.identifier.hashedEmail = [self.email toSha1];`